instructor/compiler.py

```diff
@@ -347,17 +337,17 @@ def primary_anti_body(assignment):
         for band in a.bands.all():
             if western_blot.has_whole_cell_lysate:
```

Comment:
I honestly think this 'marks' list is not needed at all. It is useful only if I want to set a band for this particular antibody to appear for all samples. I don't think such case is common, and the assignment builder does not provide such option to the instructor.